tools-local/tasks/FinalizeBuild.cs

```diff
@@ -97,6 +97,8 @@ public override bool Execute()
                     CopyBlobs($"{Channel}/Installers/{SharedFrameworkNugetVersion}", $"{Channel}/Installers/Latest/");
 
                     CopyBlobs($"{Channel}/Installers/{SharedHostNugetVersion}", $"{Channel}/Installers/Latest/");
+                    
+                    CopyBlobs($"{Channel}/Checksums/{SharedFrameworkNugetVersion}", $"{Channel}/Checksums/Latest/");
```

Comment:
Yeah I was wondering whether we should have an explicit checksum storage account, will ping you